src/components/EditUserMetadata.jsx

```diff
@@ -123,12 +130,19 @@ export default function EditUserMetadata({
                 : omit(fieldValues, ['email']);
               const patchValues = map(
                 patchFieldValues,
-                (value, key) => ({
+                (value, key) => {
+                  if(key === twitterMetadataKey){
```

Comment:
looks like you need to run the formatter (`npm run format`)

Reply:
Done